torch/_export/passes/add_runtime_assertions_for_constraints_pass.py

```diff
@@ -254,7 +269,11 @@ def sym_size_cb(proxy, assert_msg, dim):
             cb(proxy=ret, assert_msg=f"{ret.node}" + msg)
         return ret
 
-    def _assert_range_constraint(self, proxy, lower, upper, assert_msg):
+    def _assert_range_constraint(self, *, proxy, lower, upper, assert_msg, symbol):
+        if symbol in self.cached_assertions:
```

Comment:
Are we sure we will only have one `(lower, upper)` per symbol?

Reply:
Currently `self.range_constraints` is map from symbol to `RangeConstraint` so there is at mostly one (lower, upper) tuple per symbol? Let me add some comment to make this more explicit.

Reply:
OK.